profiles/nuvla.json

```diff
@@ -0,0 +1,129 @@
+{
+  "name": "Nuvla",
+  "revision": "2015-10-13",
+  "vendor_verified": "2015-10-13",
+  "url": "http://nuv.la/login",
+  "status": "production",
+  "status_since": "2012-01-01",
+  "type": "Generic",
+  "hosting": {
+    "public": true, "private": true
+  },
+  "pricing": [
+    {
+      "model": "metered",
+      "period": "monthly"
+    },
+    {
+      "model": "free"
+    }
+  ],
+  "scaling": {
+    "vertical": true, "horizontal": true, "auto": false
+  },
+  "runtimes": [
+    { "language": "python",     "versions": [ "*.*" ] },
+    { "language": "ruby",       "versions": [ "*.*" ] },
+    { "language": "bash",       "versions": [ "*.*" ] },
+    { "language": "powershell", "versions": [ "*.*" ] }
```

Comment:
That's why I suggested a `description` field to set up the context of each PaaS :blush: The term "PaaS" currently applies to such a broad range of platforms, that is indeed difficult to compare.
In our case `bash` (and thus `PowerShell`) can be considered as runtimes.